flight/mcu/lib/teensyShared/telemetry.cpp

```diff
@@ -16,6 +16,15 @@
 // #define SERIAL_PLOTTING
 
 #include <telemetry.h>
+#include <limits>
+
+
+template<typename T>
+T inv_convert_range(float val, float range){
```

Comment:
`T inv_convert_range(float val, float inp_range){`
I think changing the name of the variable would make quite a bit clearer. My first interpretation of `range` was that it was the output's range, sort of a like a lerp function.